src/config.js

```diff
@@ -0,0 +1,18 @@
+import Utils from './utils';
+
+if (!window.prismic || !window.prismic.endpoint) throw new Error('Prismic.js requires window.prismic.endpoint');
+
+const PRISMIC_ENDPOINT = window.prismic.endpoint.replace(/\.cdn\.prismic\.io/, '.prismic.io');
+const matches = PRISMIC_ENDPOINT && PRISMIC_ENDPOINT.match(new RegExp('https?://([^/]*)'));
+
+export default matches ? {
+  baseURL: matches[0],
+  editorTab: matches[1],
+  corsLink: Utils.iFrame(`${matches[0]}/previews/messenger`),
+  location: {
+    origin: window.location.origin,
+    hash: window.location.hash,
+    pathname: window.location.pathname,
+    search: window.location.search,
+  },
+} : null;
```

Comment:
that's ugly to export a null module. I'd prefer that we simply stop the code as soon as the endpoint doesn't match instead of putting that in a potentially null module and check if it's not in another 2 places.